SECURITY.md

```diff
@@ -285,6 +285,74 @@ public <ReqT, RespT> ServerCall.Listener<ReqT> interceptCall(ServerCall<RespT> c
 
 [Mutual authentication]: http://en.wikipedia.org/wiki/Transport_Layer_Security#Client-authenticated_TLS_handshake
 
+## Troubleshooting
+
+If you received an error message "ALPN is not configured properly" or "Jetty ALPN/NPN has not been properly configured", it most likely means that:
+ - ALPN related dependencies are either not present in the classpath
+ - or that there is a classpath conflict
+ - or that a wrong version is used due to dependency management.
+
+### Netty
+If you aren't using gRPC on Android devices, you are most likely using `grpc-netty` transport.
+
+Find the dependency tree (e.g., `mvn dependency:tree`), and look for versions of:
+ - `io.grpc:grpc-netty`
+ - `io.netty:netty-codec-http2`
+ - `io.netty:netty-tcnative-boringssl-static:jar` 
+
+If `netty-tcnative-boringssl-static` is missing, then you either need to add it as a dependency, or use alternative methods of providing ALPN capability by reading the *Transport Security (TLS)* section carefully.
+
+If you have both `io.grpc:grpc-netty` and `io.grpc:grpc-okhttp`, you may also have issues. Remove `grpc-okhttp` if you are not on Android.
```

Comment:
I don't think this is true.

Reply:
@zpencer ?

Reply:
@ejona86 is right, looks like this was a mistake. It turns out that ManagedChannelBuilder prefers netty, except in the case of appengine+jdk7 and the case of android. If users create the okhttp channel via `OkHttpChannelBuilder` then there is no problem. If ManagedChannelBuilder selects netty, then the usual prerequisite of netty-tcnative applies and may lead to this ALPN error.